devops/actions/llvm_test_suite/action.yml

```
@@ -83,6 +83,11 @@ runs:
       if [ -e /runtimes/oneapi-tbb/env/vars.sh ]; then
         source /runtimes/oneapi-tbb/env/vars.sh;
       fi
+      # TODO remove workaround of FPGA emu bug
+      mkdir icd
```

Shouldn't we check if `icd` directory exists already?
```suggestion
      mkdir -p icd
```